src/OrbitGl/GlSlider.cpp

```diff
@@ -7,6 +7,11 @@
 #include <GteVector.h>
 #include <math.h>
 
+#ifdef ORBIT_GL_USE_QT
```

Comment:
What is the purpose of the `ifdef`? Are we going to compile this also without the preprocessor macro set?

Reply:
I've added this since the current linking against QT is quite locally and it would be still easily possible to compile OrbitGl without this by just undefining ORBIT_GL_USE_QT. I'm not really sure if we want this, though, I can remove it if we think it sets a bad example for future, more complex uses of QT, wdyt?

Reply:
I tend to vote for removing it. To me this is similar to commented code - we only ever use one version of it while still having the mental overhead of being an optional thing where both options need to be considered. If we really want to get rid of it we can just revert the commit.

Reply:
As commented above - I removed this now.